src/shared/services/limitStore.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ LimitStore.put('colorado', 'rocky');
+ ["rocky"]
+ LimitStore.put('colorado', 'mountain');
+ ["rocky", "mountain"]
+ LimitStore.put('colorado', 'national');
+ ["mountain", "national"]
+ LimitStore.put('colorado', 'park');
+ ["national", "park"]
+ */
+const LIMIT = 2;
+const store: Record<string, Array<unknown>> = {};
+
+export function limitStorePut<T>(key: string, value: T) {
+  if (store[key] === undefined) {
+    store[key] = [null, null];
+  }
+  store[key].push(value);
+  if (store[key].length == LIMIT + 1) {
```

Comment:
consider using strict equality
```suggestion
  if (store[key].length === LIMIT + 1) {
```

Reply:
Done